core/src/main/java/org/apache/carbondata/core/datastore/columnar/BlockIndexerStorage.java

```diff
@@ -50,7 +83,7 @@
    *
    * @param rowIds
    */
-  protected Map<String, short[]> rleEncodeOnRowId(short[] rowIds) {
+  protected void rleEncodeOnRowId(short[] rowIds) {
```

Comment:
since you are modifying this, can you change the function name to a more intuitive name?

Reply:
done